File: 0201_0300/0294/0294.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    k1, l1, m1, k2, l2, m2 = (int(x) for x in f.read().split())
    a = min(k1 * (100 - l1), k2 * (100 - l2)) // 100
    q.write(str((k1 - a) * m1 + (k2 - a) * m2))

File: 0201_0300/0294/0294.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int k1, l1, m1, k2, l2, m2, a;

	fscanf(f, "%d%d%d%d%d%d", &k1, &l1, &m1, &k2, &l2, &m2);
	a = (k1 * (100 - l1) < k2 * (100 - l2) ? k1 * (100 - l1) : k2 * (100 - l2)) / 100;
	fprintf(q, "%d", (k1 - a) * m1 + (k2 - a) * m2);
	return 0;
}
